packages/runtime-core/__tests__/apiCreateComponent.spec.tsx

```diff
@@ -5,7 +5,7 @@ import { h } from '../src/h'
 
 // mock React just for TSX testing purposes
 const React = {
-  createElement: () => {}
+  createElement: () => { }
```

Comment:
There seems to be a bunch of unnecessary style/whitespace changes in this file. Could you please remove them?